### PR TITLE
Add e2e test for certman

### PR DESCRIPTION
**Pull request by wanghaoran1988**

### CODE REVIEW

**Comment by wanghaoran1988** [comment]
@clcollins Could you take a look?
/cc @c-e-brumm

File: test/operators/certman.go [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+	. "github.com/onsi/gomega"
+	"github.com/openshift/osde2e/pkg/helper"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var _ = ginkgo.Describe("[OSD] Certman Operator", func() {
+	h := helper.New()
+	secretName := h.ClusterName + "-" + "primary-cert-bundle-secret"
+	ginkgo.Context("certificate secret  should be applied when cluster installed", func() {
```
Comment (meowfaceman) [inline test/operators/certman.go]: nit: Double space between "secret" and "should."

**Comment by c-e-brumm** [comment]
Code looks good +1 on that. If I understand the intention then we're checking a provisioned test cluster for the secrets and certs that we expect Hive to have passed it during creation. Just double checking that osde2e clusters are built by Hive. If that's accurate then this test looks good to confirm certman was working.

**Comment by wanghaoran1988** [comment]
> Just double checking that osde2e clusters are built by Hive. If that's accurate then this test looks good to confirm certman was working.

Yes, the clusters are built by Hive.

**Pull request merged**